frame/babe/src/mock.rs

```diff
@@ -52,6 +52,7 @@ frame_support::construct_runtime!(
 		UncheckedExtrinsic = UncheckedExtrinsic,
 	{
 		System: frame_system::{Pallet, Call, Config, Storage, Event<T>},
+		Authorship: pallet_authorship::{Pallet, Call, Storage, Inherent},
```

Comment:
This is not used as far as I can see. If so, You should remove the `impl` block rather than adding it here.

Reply:
pallet authorship is used in the equivocation handler (equivocation handler used in the mock requires pallet authorship)